includes/class-wc-payments-order-service.php

```diff
@@ -345,6 +345,11 @@ public function mark_payment_dispute_closed( $order, $charge_id, $status ) {
 			return;
 		}
 
+
+		// Order `completed` and `refunded` emails should both be blocked when disputes are closed.
+		add_filter( 'woocommerce_email_enabled_customer_completed_order', '__return_false' );
+		add_filter( 'woocommerce_email_enabled_customer_refunded_order', '__return_false' );
```

Comment:
I considered only adding the specific filters inside the `if`, but it seemed more readable to me to add and remove both at the same time.